core/src/avm2/traits.rs

```diff
@@ -287,6 +410,10 @@ impl<'gc> Trait<'gc> {
         &self.kind
     }
 
+    pub fn metadata(&self) -> Vec<TraitMetadata<'gc>> {
+        self.metadata.clone()
```

Comment:
If we _do_ decide store it here, we might wanna store it behind a Gc, to minimize Trait struct size and vec copying.

Reply:
Stored behind an `Option<Box<`